faq/faq.md

```diff
@@ -0,0 +1,351 @@
+such an error would require rewriting the history, causing all contributors to
+have to re-download the new trunk.
+
+## Can I use DEP5 to declare licensing and copyright? {#dep5}
+
+See next question.
+
+## Can I bulk-license whole directories? {#bulk-license}
+
+DEP5
+
+## Can I license only a part of a file as being under a different license? {#partial-license}
+
+The short answer is that yes, you can, but no, there is no standard way for
+REUSE to recognize this. If you have a small segment of a file that is licensed
+differently, you should list that license under a separate
+`SPDX-License-Identifier` tag in the header.
+
+You can use your own comments to specify which segment is separately licensed.
+For instance: "The class Foo is copied from project Bar and licensed under MIT.
+All changes are licensed under GPL-3.0-or-later."
+
+A possible way to circumvent the problem is to extract the segment from the
+file, and to keep it in its own file.
+
+## How do I properly declare multi-licensing? {#multi-licensing}
+
+You should always include all licenses in the `LICENSES/` directory.
+
+The correct SPDX license expression that applies to the file depends on the
+intent. If all the code within is licensed under multiple licenses, and the
+licensee can choose under which license they consume the work, use
+`SPDX-License-Identifier: MPL-1.1 OR GPL-2.0-or-later OR LGPL-2.1-or-later`, as
+parts of Firefox do.
+
+If all the code within the file is licensed under multiple licenses, and the
+user must comply with all licenses simultaneously, use
+`SPDX-License-Identifier: LGPL-2.0-or-later AND AML`, as can be found in Simple
+DirectMedia Layer (SDL).
+
+If all the code within the file is licensed under either one license or another
+(for instance, all code is under GPL-2.0-only, but one function is under MIT),
+use separate tags `SPDX-License-Identifier: GPL-2.0-only` and
+`SPDX-License-Identifier: MIT`.
+
+You can read more about SPDX expressions [on the SPDX
+wiki](https://wiki.spdx.org/view/LicenseExpressionFAQ).
+
+## How to deal with MIT/BSD licenses which include copyright information themselves? {#mit-bsd}
+
+TODO: Figure this one out
+
+## Should I move my existing LICENSE or COPYING file? {#move-license}
+
+If you use multiple licenses in your project, we recommend that you put all of
+your licenses in the `LICENSES/` directory.
+
+If you only have a single license file, see [the next
+question](#single-license).
+
+## I only have a single license file. Should I still create a LICENSES directory? {#single-license}
+
+You do not strictly need to put your license in the `LICENSES/` directory,
+though we recommend that you do. If you want to keep your license in a single
+file (`LICEN[CS]E`, `COPYING`, `COPYRIGHT`), you should edit the file to signal
+what licence you are actually using.
+
+Preface your license with the following header:
+
+```
+Valid-License-Identifier: [identifier]
+License-Text:
+
+[license text]
+```
```

Comment:
I’m wondering if we can obsolete the `Valid-License-Identifier` and `License-Text` fields by making REUSE even closer to SPDX. Ping me, if I forget to come back to this. This could be a big simplification of the spec.

Reply:
How do you suggest making REUSE closer to SPDX, short of mandating `LICENSES/[identifier].txt`?

Reply:
As discussed in the Spec (discussion on PR #23), I think we actually could mandate `LICENSES/[spdx-id].[extension]`

Reply:
Closing this for #27